app/services/katello/pulp/repository.rb

```diff
@@ -247,6 +251,32 @@ def index_content_units
           content_type.pulp2_service_class
         end
       end
+
+      def proxy_options
+        if repo.root.http_proxy_policy == RootRepository::NO_DEFAULT_HTTP_PROXY
+          return {}
```

Comment:
i think its actually more than returning nothing, i think we should return empty string for proxy 'host'.  Also the same below

Reply:
and the reason is that it makes this change independent of the installer dropping in a 'global' proxy config for pulp2, that can continue to happen, but if we use empty string, it'll be ignored.  (eventually we'll remove it though)